azure-communication-ui/calling/README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Latest Release
 
-- [1.11.0 release](https://github.com/Azure/communication-ui-library-android/releases/tag/calling-v1.11.0)
+- [1.14.2 release](https://github.com/Azure/communication-ui-library-android/releases/tag/calling-v1.14.2)
```

Comment:
I wonder if instead of direct linking to a version we just link to the release page so we don't need to keep updating this

Reply:
It also needs to be updated in the usage example anyway...